src/images/garden-wordmark.svg

```diff
@@ -0,0 +1,18 @@
+                <path d="M74.7166449,13.9975466 L81.2395602,13.9975466 L81.2395602,18.7208266 C82.9228932,15.6404266 85.8687259,13.9975466 90.1822667,13.9975466 L93.1280994,13.9975466 L93.1280994,20.0556667 L89.1301836,20.0556667 C83.5541431,20.0556667 81.3447686,22.8280267 81.3447686,28.9888268 L81.3447686,43.7747469 L74.7166449,43.7747469 L74.7166449,13.9975466 L74.7166449,13.9975466 Z" id="Path"></path>
+                <path d="M95.1270573,28.8861467 C95.1270573,19.3369066 102.386431,13.3814666 110.382262,13.3814666 C114.380178,13.3814666 118.272886,15.1270266 120.48226,17.7967066 L120.48226,0.341106444 L126.899967,0.341106444 L126.899967,43.7747469 L120.48226,43.7747469 L120.48226,39.6675469 C118.167677,42.5425869 114.27497,44.3908269 110.277054,44.3908269 C102.596847,44.3908269 95.1270573,38.4353869 95.1270573,28.8861467 Z M120.903094,28.8861467 C120.903094,23.5467867 116.799969,19.2342266 111.223929,19.2342266 C105.753097,19.2342266 101.544764,23.4441067 101.544764,28.8861467 C101.544764,34.3281868 105.753097,38.5380669 111.223929,38.5380669 C116.799969,38.5380669 120.903094,34.2255068 120.903094,28.8861467 Z" id="Shape" fill-rule="nonzero"></path>
+                <path d="M131.634341,28.9888268 C131.634341,20.2610267 138.472882,13.3814666 147.520796,13.3814666 C156.568711,13.3814666 163.196835,19.7476267 163.196835,28.5781067 L163.196835,31.0424268 L137.94684,31.0424268 C138.788506,35.6630268 142.470797,38.6407469 147.731213,38.6407469 C151.834337,38.6407469 155.095795,36.4844668 156.673919,33.3013868 L162.039543,36.1764268 C159.304127,41.1050669 154.569753,44.2881469 147.731213,44.2881469 C138.157257,44.3908269 131.634341,37.7166268 131.634341,28.9888268 Z M138.262465,25.7030667 L156.568711,25.7030667 C155.621836,21.3905067 152.25517,19.0288666 147.520796,19.0288666 C142.996839,19.0288666 139.524965,21.8012267 138.262465,25.7030667 Z" id="Shape" fill-rule="nonzero"></path>
+                <path d="M167.931209,13.9975466 L174.348916,13.9975466 L174.348916,17.8993866 C176.453082,15.1270266 179.71454,13.4841466 183.712455,13.4841466 C191.392662,13.4841466 196.127036,18.4127866 196.127036,26.4218267 L196.127036,43.7747469 L189.498912,43.7747469 L189.498912,26.9352267 C189.498912,22.0065867 186.973913,18.9261866 182.134331,18.9261866 C177.715582,18.9261866 174.348916,22.1092667 174.348916,27.3459467 L174.348916,43.6720669 L167.826,43.6720669 L167.826,13.9975466 L167.931209,13.9975466 Z" id="Path"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
This should be getting sourced from `svg-icons`. (Maybe that is planned as a follow-up)

Reply:
Didn't realize the wordmark was already added. Now provided from svg-icons.